testConfig.py

```
@@ -94,6 +97,24 @@ def _construct_capabilities_pod(
+    arg = common.structparse_check_strdict(pctx.arg, pctx.yamlpath)
+    result: dict[str, str] = {}
+    for key, value in arg.items():
+        if not key:
+            raise pctx.value_error("label key cannot be empty")
+        if value is None:
+            result[key] = ""
+        elif isinstance(value, str):
+            result[key] = value
+        else:
+            raise pctx.value_error(
+                f"expects string label values but got {repr(value)}",
```

Ruff (0.15.20)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)